core/src/test/scala/co.blocke.scalajack/v4/SimpleTests.scala

```diff
@@ -7,10 +7,17 @@ import scala.language.postfixOps
 import scala.util.Try
 import org.joda.time.{ DateTime, DateTimeZone }
 import org.joda.time.format.DateTimeFormat
+import scala.reflect.runtime.universe.{ Type, typeOf }
+import BijectiveFunctions.fullNameToType
 
 class SimpleTestSpec extends FunSpec with GivenWhenThen with BeforeAndAfterAll {
   val sjJS = ScalaJack()
 
+  val hintMod = new HintModifier {
+    def apply(rawHint: String) = fullNameToType.apply("co.blocke.scalajack.test.v4." + rawHint)
+    def unapply(hintFieldType: Type) = fullNameToType.unapply(hintFieldType).split('.').last
```

Comment:
This almost suggests that there is another `BijectiveFunction` that we should write. Maybe something like:
```
def unqualifiedClassNameIn(packageName: String): BijectiveFunction[String, String]
```
Then maybe you could say something like `fullNameToType andThen unqualifiedClassNameIn("co.blocke.scalajack.test.v4")`